labellab-flask/api/serializers/team.py

```diff
@@ -0,0 +1,15 @@
+from marshmallow import Schema, fields
+
+from api.extensions import db, ma
+from api.serializers.projectmember import ProjectMemberSchema
+
+class TeamSchema(ma.ModelSchema):
+    """
+    Serializer class for team
+    """
+        
+    id = fields.Int(dump_only=True) 
+    teamname = fields.Str()
```

Comment:
team_name

Reply:
Yes these were suggested by @Tushar19varshney  in #447 I have referenced it there.